rke/templates/nginx-ingress.go

```diff
@@ -960,7 +960,9 @@ spec:
                     - windows
                 - key: node-role.kubernetes.io/worker
                   operator: Exists
+      {{if eq .NetworkMode "hostNetwork"}}
       hostNetwork: true
+      {{end}}
```

Comment:
does this need to be `{{- end }}` to avoid adding a newline

Reply:
Cleaned up the template.